src/_pytest/main.py

```diff
@@ -241,6 +241,16 @@ def pytest_addoption(parser: Parser) -> None:
         ),
     )
 
+    """Check basetemp dir
```

Comment:
Docstrings in python fo into the definition

Reply:
It also references a non-existent argument. It's not a docstring really...